app/controllers/master/events/imports_controller.rb

```diff
@@ -13,6 +15,7 @@ def create
   private
 
   def check_shared_secret
-    head(403) unless params[:secret] == Events.shared_secret
+    permitted_params = params.permit(%i[secret])
+    head(403) unless permitted_params[:secret] == Events.shared_secret
```

Comment:
This will break in rails 5.1
```suggestion
    permitted_params = params.permit(:secret).to_h
    head(403) unless permitted_params[:secret] == Events.shared_secret
```